mubench.reviewsite/bootstrap/renderer.php

```diff
@@ -31,6 +34,31 @@
 
     $markdown_parser = new Parsedown();
     $markdown_parser->setBreaksEnabled(True);
+    $exp_det_reviewstate = [];
```

Comment:
This logic should not be here. It should be in some file (class?) dedicated to the menu logic. I imagine that this code here calls a function/creates an object, such that the template can make queries like `$o->hasOpenReviews($detector, $experiment)` to decide what to display.

The logic needs tests and it should be split into multiple functions. There's way to much nesting here.